presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -298,7 +298,7 @@ public static Block split(@SqlType("varchar(x)") Slice string, @SqlType(Standard
                  index++;
             }  
           
-        } else if (delimeter.length()==0) {
+        } else if (delimeter.length()>0) {
```

Comment:
What's the purpose of this change? If it's related to the commit to allow empty string as delimited, please squash it it with that commit.